swap/src/bin/swap.rs

```diff
@@ -151,8 +151,8 @@ async fn main() -> Result<()> {
             table.printstd();
         }
         Command::Resume {
-            swap_id,
-            alice_multiaddr,
+            swap_id: SwapId { swap_id },
+            seller_addr: SellerAddr { seller_addr },
```

Comment:
To make this less verbose, we could introduce another level of structs between the actual structopt structs and what is handled by `main.rs`.

This would also be inline with resolving https://github.com/comit-network/xmr-btc-swap/issues/360. We [had that at some point](https://github.com/comit-network/xmr-btc-swap/blob/948cc99e9d87dfd710fa3cae15bb777710669dbd/swap/src/cli/command.rs) but that PR never got merged.

Reply:
https://github.com/comit-network/xmr-btc-swap/pull/490/commits/9f5a686d4897797f1fd496dd30499ea165334657